content/22.vaccines.md

```diff
@@ -316,7 +315,37 @@ In a phase III randomized, observer-blinded, placebo-controlled clinical trial i
+The approach utilizes _Nicotiana benthamiana_, an Australian relative of the tobacco plant, as an upstream bioreactor [@doi:10/gjn28x; @doi:10.1038/s41591-021-01370-1; @doi:10.1038/s41591-021-01370-1].
+Specifically, the _S_ gene from SARS-CoV-2 in its prefusion conformation is inserted into a bacterial vector (_Agrobacterium tumefaciens_) that then infects the tobacco plant cells [@doi:10/gjn28x; @doi:10.1038/s41591-021-01370-1].
+Expression of the S glycoprotein causes the production of VLPs composed of S trimers anchored in a lipid envelope that accumulate between the plasma membrane and the cell wall of the plant cell [@doi:10.1038/s41591-021-01370-1].
+Because these VLPs do not contain the SARS-CoV-2 genome, they offer similar advantages to while mitigating the risks of whole-virus vaccines [@doi:10/gjn28x; @doi:10.1038/s41591-021-01370-1].
+
+In the Phase I study, Covifenz was administered to 180 Canadian adults 18-55 years old as two doses, 21 days apart, with three different dosages evaluated [@doi:10.1038/s41591-021-01370-1].
+This study reported that when the VLPs were administered with an adjuvant, the vaccine elicited a neutralizing antibody that was significantly (approximately 10 times) higher than that in convalescent sera  [@doi:10.1038/s41591-021-01370-1].
+They also reported a cellular immune response was induced.
+Based on these findings, phase II and III trials began.
+
+The findings of the phase III trial were published a few months later in the _New England Journal of Medicine_ [@url:10.1056/NEJMoa2201300].
+This study examined 24,141 adults assigned to the treatment and control conditions at a 1:1 ratio between March and September of 2021.
+Participants were recruited from countries in North and South America, as well as the United Kingdom.
+Approximately 10,000 individuals in each condition completed both doses and were evaluated in the per-protocol population.
+Data submitted by Medicago and GSK to Health Canada, the vaccine was approved for use in adults ages 18 to 65 in February 2022 [@url:https://www.gsk.com/en-gb/media/press-releases/medicago-and-gsk-announce-the-approval-by-health-canada-of-covifenz].
+
+Plant-based expression systems such as this are relatively new [@doi:10.1038/s41591-021-01370-1] but are likely to offer unparalleled feasibility at scale given the speed and low-cost associated with the platform [@doi:10.1111/j.1467-7652.2008.00384.x].
+Additionally, it can be stored at 2 to 8&#176;C.<!--Check this renders correctly as degree symbol-->
+However, the worldwide footprint of Covifenz, and of VLP-based technologies against SARS-CoV-2 broadly, remains small, with only {{owid_VLP_count}} VLP vaccine has been approved for distribution in {{owid_VLP_countries}} countries (Figure @fig:vlp-distrib).<!--Grammar needs update if anyone outside of Canada approves or if another VLP becomes active-->
```

Comment:
This confused me grammatically but I see your note, so ignoring.

Reply:
```suggestion
However, the worldwide footprint of Covifenz, and of VLP-based technologies against SARS-CoV-2 broadly, remains small, with only {{owid_VLP_count}} VLP vaccine approved for distribution in {{owid_VLP_countries}} countries (Figure @fig:vlp-distrib).<!--Grammar needs update if anyone outside of Canada approves or if another VLP becomes active-->
```
It was wrong! But obfuscated by all the variables. Good catch!